src/i18n/de/requests.js

```diff
@@ -1,7 +1,7 @@
 export default {
-  request: 'Gesuch',
-  myRequests: 'Meine Anzeigen',
-  noRequestsCreated: 'Du hast noch keine Anzeigen erstellt.',
+  request: 'Hilfseintrag',
```

Comment:
Maybe "Hilfsanfrage" is even more descriptive because "Eintrag" could also mean that you offer your help.

Reply:
Indeed